### ISSUE TITLE
[Feature request] Ability to highlight multiple elements at once

### ISSUE THREAD

**Issue by codemunkie15** [IGreatlyDislikeJavascript/bootstrap-tourist#48]
I have a bootstrap grid and I want to highlight two of the three columns for a tour step. I tried to give the columns a common class and use that as the element but the tour step appeared twice, one for each column. I also tried to wrap the two columns in a div and give that an id to use but that breaks the bootstrap grid.

Is this possible to implement? Maybe a separate selector for the highlight?

Thanks

**Comment by IGreatlyDislikeJavascript** [comment]
Yes, you can implement this (incidentally with the same solution as for your other report). I'm a little pressed for time, but have a look at `backdropOptions.animation` per-step options in the documentation.

Overriding these options and providing your own function allows you to position the highlight yourself, instead of Tourist doing it. These options are really intended for animation, but as they completely override all transition and positioning (hence the requirement to call `step.fnPositionHighlight();`), you can use them for anything.

If you have trouble implementing it, or if you'd like this as a formal feature (I think it's an edge case though), please let me know.
